Userland/Libraries/LibC/fcntl.cpp

```diff
@@ -104,6 +104,12 @@ int posix_fadvise(int fd, off_t offset, off_t len, int advice)
     return 0;
 }
 
```

Comment:
missing spec link here at top of the function

Reply:
The problem here is that the opengroup man page and the linux man page disagree in point

See that unanswered discord question:
https://discord.com/channels/830522505605283862/830525093905170492/990694704008007680